### ISSUE TITLE
Add Light Mode/Honor System Setting for Dark Mode

### ISSUE THREAD

**Issue by wmehilos** [redcanaryco/mac-monitor#24]
**At a high level -- can you summarize your request?**
The app should have a light mode for the UI, and should have an option to honor the user's system preference for Light/Dark mode. 

**What is the current alternative solution?**
There does not appear to be a way to change the appearance of the app to anything but dark mode. 

**Are there "In-the-Wild" threats or corresponding ATT&CK techniques that exist for which this telemetry would be helpful?**
N/A

**Anything else?**
Nope! 🙂

**Comment by Brandon7CC** [comment]
Hi @wmehilos!

I completely sympathize with this request, but due to my own "design requirements" for a light mode this task is too technically complex for me to implement with my current backlog in-mind. 

I'll keep it noted that the community would appreciate this feature in the future though 😄
